govc/fields/add.go

```diff
@@ -37,19 +38,22 @@ func init() {
 func (cmd *add) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
 	cmd.ClientFlag.Register(ctx, f)
-}
 
-func (cmd *add) Process(ctx context.Context) error {
-	if err := cmd.ClientFlag.Process(ctx); err != nil {
-		return err
-	}
-	return nil
+	f.StringVar(&cmd.kind, "type", "", "Managed object type")
 }
 
 func (cmd *add) Usage() string {
 	return "NAME"
 }
 
+func (cmd *add) Description() string {
+	return `Add a custom field type with NAME.
+
+Examples:
+  govc fields.add my-field-name
```

Comment:
Hi @dougm,

Would it be useful to add a comment that if the `type` is omitted, a global field is created?

```suggestion
  govc fields.add my-field-name # adds a field to all managed object types
```